### PR TITLE
Add ng-dropzone w/ git auto-update via single package.json

### LINKED ISSUE #11476: [Request] Add ng-dropzone
**Library name:** ng-dropzone
**Git repository url:** https://github.com/thatisuday/ng-dropzone
**npm package**: [ngdropzone](https://www.npmjs.com/package/ngdropzone)
**License:** Apache-2.0
**Official homepage:** http://www.dropzonejs.com
**Note:** Please note that **gz** files should be excluded in commits and auto-update config

=====================

For beginners who is interested in picking up this issue:

In this issue, we are going to host a new web front-end library on CDNJS, so that web developer can easily use it on our free CDN without additional download/upload process 🚀 .

If you'll like to work locally, I'll suggest you start pulling cdnjs repository with sparseCheckout: https://github.com/cdnjs/cdnjs/blob/master/documents/sparseCheckout.md, since there are so many files in this repo, without sparseCheckout, the whole process will be slow and will use more than 80GB disk space, using git command line can learn more skills and experience, if you don't like that, instead, you can also just do it on GitHub, it's more convenient, but it may be a little bit harder to fix problems on GitHub gui.

Here are the docs about how to add a library:
 - https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING.md#d-adding-a-new-library-with-its-assets
 - https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING.md#e-adding-a-new-library-by-a-single-packagejson

We'll also ask to setup auto-update config so that the added libraries can be updated to the latest version easily:
 - https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md

For beginners who is interested in join and help add this library, please feel free to tell me what kind of help you'll need, we'll help you send the valid pull request, thanks 😄

### PR DESCRIPTION
**Pull request by prathap0611**
close #11476, cc @thatisuday

Pull request for issue: #11476 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/thatisuday/ng-dropzone
 * Official website (optional, not the repository): http://www.dropzonejs.com
 * NPM package url (optional): https://www.npmjs.com/package/ngdropzone
 * License and its reference: Apache-2.0
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 7
   - Count of stars: 71
   - Count of forks: 18
 * NPM download stats (optional):
   - Downloads in the last day: 69
   - Downloads in the last week: 262
   - Downloads in the last month: 1114

# Essential checklist
 * [ ] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/ng-dropzone/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "ng-dropzone",
+  "filename": "ng-dropzone.min.js",
+  "description": "AngularJS directive for Dropzone, an easy to use drag'n'drop file upload library.",
+  "homepage": "http://www.dropzonejs.com",
+  "keywords": [
```
Comment (extend1994) [inline ajax/libs/ng-dropzone/package.json]: Maybe keep the original two keywords: "ngDropzone" and "dropzone"? Ref: https://github.com/thatisuday/ng-dropzone/blob/master/bower.json#L9

Comment (prathap0611) [inline ajax/libs/ng-dropzone/package.json]: @extend1994  Sure I'll replace the keywords as specified here.

**Comment by sufuf3** [comment]
@prathap0611 May I ask you how do you send this pull request? via GitHub website or use git command?
I think the commit should squash to one commit, if you use git command, we can help you how to do this.
But if you use GitHub website, it can't squash the commit. I can help you squash to one commit.

Thank you for your contribution.

**Comment by prathap0611** [comment]
@sufuf3 I am using git command. I'll squash the commits into single commit and push again. Thank you for your support.

**Comment by prathap0611** [comment]
@sufuf3 I have squashed and pushed a commit.

**Comment by sufuf3** [comment]
Good 
I think this PR is fine

@extend1994 
Please help review this PR, Thank you.

**Comment by PeterDaveHello** [comment]
@extend1994 @sufuf3 @dakshshah96 @prathap0611:

The second line of every commit message should always be empty, the appending `close #11476, cc @thatisuday` should start with 3 line.

**Pull request merged**